specification/columns/pricingquantity.md

```diff
@@ -16,7 +16,11 @@ Pricing Quantity
 
 The volume of a given SKU associated with a *resource* or *service* used or purchased, based on the Pricing Unit.
 
-## Content constraints
+## Usability Constraints
+
+**Aggregation:** When aggregating Pricing Quantity for commitment utilization calculations, it's important to exclude *commitment discount* purchases (i.e. when *ChargeCategory* is "Purchase") that are paid to cover future eligible charges (e.g., *Commitment Discount*). Otherwise, when accounting for all upfront or accrued purchases, it's important to exclude *commitment discount* usage (i.e. when *ChargeCategory* is "Usage"). This exclusion helps prevent double counting of these quantities in the aggregation.
```

Comment:
Consider omitting "for commitment utilization calculations" in the case of PricingQuantity.

```suggestion
**Aggregation:** When aggregating Pricing Quantity, it's important to exclude *commitment discount* purchases (i.e. when *ChargeCategory* is "Purchase") that are paid to cover future eligible charges (e.g., *Commitment Discount*). Otherwise, when accounting for all upfront or accrued purchases, it's important to exclude *commitment discount* usage (i.e. when *ChargeCategory* is "Usage"). This exclusion helps prevent double counting of these quantities in the aggregation.
```

Reply:
Agree with the consensus from TF-1 that it's not clear what other purpose exists than commitment utilization in this instance.